src/components/Nav/Nav.css

```diff
@@ -0,0 +1,25 @@
+.nav-bar {
+  position:absolute;
+  width: 100%;
+  height: 75px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 5;
+  background-color: rgba(0, 0, 0, 0.438)
+}
+.nav-bar__home {
```

Comment:
The most minor of nitpicks here but I'm noticing that there's not a space above this line. Which makes me think: do we have a linter set up? If not, we should probably get that going. I'll make an issue for it so we don't forget.

Reply:
yes we should definitely get a linter set up. I'm bad at spacing